backend/danswer/document_index/vespa/index.py

```diff
@@ -187,6 +199,88 @@ def ensure_indices_exist(
                 f"Failed to prepare Vespa Danswer Index. Response: {response.text}"
             )
 
+    @staticmethod
+    def register_multitenant_indices(
```

Comment:
If it's on kubernetes and it spins up another container for the api server, won't it recreate the indices, so if someone sets a new non-default model, it will get wiped? We don't need to support this for now but maybe we should make a note of it.

On the branch we deploy for the cloud, we might want to just choose a model for them and not let them specify their own.